src/modules/common/homeassistant.js

```diff
@@ -70,6 +70,7 @@ export class Homeassistant {
     }
 
     getServices(callback) {
+        console.error(callback);
```

Comment:
I think this line was for debugging purposes? :D
```suggestion
```

Reply:
Yeah, it was just for debugging. I already knew this, but it was just the RAW Json so for debugging I used the formatted Console output.